assets/app/views/images.html

```diff
@@ -10,30 +10,48 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>
-      <div ng-repeat="imageStream in imageStreams">
-        <div class="tile" ng-repeat="tag in imageStream.status.tags">
+      <div ng-repeat="imageStream in imageStreams track by (imageStream | uid)" class="tile">
+        <h2>
+          {{imageStream.metadata.name}}
+          <small class="tile-details">Created
+            <relative-timestamp timestamp="imageStream.metadata.creationTimestamp"></relative-timestamp></small>
+        </h2>
+        <div ng-if="imageStream.spec.dockerImageRepository" class="small">Docker image repository: {{imageStream.spec.dockerImageRepository}}</div>
+        <!-- Print out missing status tags with a warning. -->
+        <div ng-repeat="specTag in missingStatusTagsByImageStream[imageStream.metadata.name] | orderBy : 'name' track by specTag.name" style="margin-top: 10px;">
           <div>
-            <span>{{imageStream.metadata.name}}:{{tag.tag}}</span>
-            <span ng-if="!tag.items.length"> &mdash; <em>No images for this tag</em></span>
-            <span ng-if="tag.items.length && tag.items[0].image">
-              <span> &mdash; </span>
-              <short-id id="{{tag.items[0].image | imageName}}"></short-id>
-              <span> created </span>
-              <relative-timestamp timestamp="tag.items[0].created"></relative-timestamp>
+            {{specTag.name}}
+            &mdash;
+            <span class="pficon-layered">
+              <span class="pficon pficon-warning-triangle"></span>
+              <span class="pficon pficon-warning-exclamation"></span>
             </span>
+            Tag could not be resolved.
```

Comment:
@ncdc is there any info recorded in the status for the UI to tell the difference between a sync not happening yet and a sync failing?